templates/custom_sso_security_manager.py

```diff
@@ -0,0 +1,18 @@
+from superset.security import SupersetSecurityManager
+
+
+class CustomSsoSecurityManager(SupersetSecurityManager):
+    def oauth_user_info(self, provider, response=None):
+        if provider == "google":
```

Comment:
What if the provider is not `google` and you get `None` out of the method? This may not be a possibility today but can it break with changes in the future? I would write this more defensively with some checks if there is such a possibility.

Reply:
+1

Reply:
If the result of the method is `None` then the superset init will not use this value but rather the default SupersetSecurityManager: https://github.com/apache/superset/blob/master/superset/initialization/__init__.py#L543. This is intentional to avoid breaks. 

As this is an extension of an existing class, the error handling for this method is handled within the superset code itself.